action/action.go

```diff
@@ -26,16 +26,21 @@ const stateful = false
 // - status
 type Action interface {
 	// Run an action, and record the status in the given claim
-	Run(*claim.Claim, credentials.Set, io.Writer) error
+	Run(*claim.Claim, credentials.Set, bundle.ImageRelocationMap, io.Writer) error
```

Comment:
I wonder if maybe this relocation map should be stored in the Claim too 🤔

Reply:
I thought about that as well and it was going to be my first approach bu I realized that the relocation map is a parameter similar to the creds file.

The current process for relocation would be:
`duffle relocate` that will move the images a new registry, this is also the step that generations the relocation map
`duffle import` that will create the claim(we need this at this point because there is not support for installing thick bundle)
`duffle install` when we pass in the registry map

The advantage I see here is that if it behaves like the creds we can maybe relocate and then upgrade an installation. If we save it in the claims and relocate the bundle the claims will be out of date and the new mapping file will have to be passed.
The downside here is that we will have to pass on all the action the mapping and we need the operator would have to store that file somewhere.

If we move it to claims we would only need to pass the mapping on import/install. The drawback is that on upgrade we would have to supply the mapping and update the claim as well.

Nevertheless I am open to the change, what do y'll think?

Reply:
I think I prefer having it in the claim, as for the user this relocation file is somehow an implementation detail.

I guess it would be simpler for the user if the `import` command creates the claim with the relocation inside, so the user won't have to persist it manually or to remember which relocation file is linked with which CNAB bundle.

And about the upgrade, I think that a user upgrading a CNAB package with another one relocated somewhere else is (or should) not be a common use case (it could be seen as installing a brand new bundle).

Reply:
As per our conversation I moved the relocation map into the claims

Reply:
That looks good. Thanks for doing that.